src/plugin.test.tsx

```diff
@@ -109,7 +114,9 @@ describe('argo-cd', () => {
 
       const rendered = render(
         <ApiProvider apis={apis}>
-          <ArgoCDDetailsWidget entity={getEntityStub} extraColumns={extraColumns} />
+          <EntityProvider entity={getEntityStub}>
+            <ArgoCDDetailsWidget extraColumns={extraColumns} />
```

Comment:
I wonder if we should take the opportunity to rename these... though I see they're re-exported with "new naming"

Reply:
I'll rename these :+1: